### ISSUE TITLE
Cloud Shell Bash - AzureRM.Netcore multiple modules installed

### ISSUE THREAD

**Issue by MaximoTrinidad** [Azure/azure-powershell#6704]
### Description

Knowing that Azure.Netcore version 0.12.0 is the latest, why the previous version 0.11.0 is also installed.
I can't use the following oneliner to force to use the latest one:

```
Azure:/
PS Azure:\> import-module AzureRM.Netcore -RequiredVersion 0.12.0
/usr/local/share/powershell/Modules/AzureRM.Netcore/0.12.0/AzureRM.Netcore.psm1 : This module requires AzureRM.Profile.Netcore version 0.12.0. An earlier version of AzureRM.Profile.Netcore is imported in the current PowerShell session. Please open a new session before importing this module. This error could indicate that multiple incompatible versions of the Azure PowerShellcmdlets are installed on your system. Please see https://aka.ms/azps-version-error for troubleshooting information.
At line:1 char:1
+ import-module AzureRM.Netcore -RequiredVersion 0.12.0
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
+ FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,AzureRM.Netcore.psm1

import-module : The module to process 'AzureRM.Netcore.psm1', listed in field 'ModuleToProcess/RootModule' of module manifest'/usr/local/share/powershell/Modules/AzureRM.Netcore/0.12.0/AzureRM.Netcore.psd1' was not processed because no valid module was found in any module directory.
At line:1 char:1
+ import-module AzureRM.Netcore -RequiredVersion 0.12.0
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : ResourceUnavailable: (AzureRM.Netcore:String) [Import-Module], PSInvalidOperationException
+ FullyQualifiedErrorId : Modules_ModuleFileNotFound,Microsoft.PowerShell.Commands.ImportModuleCommand
```

### Script/Steps for Reproduction

```powershell
Azure:/
PS Azure:\> Get-Module -ListAvailable AzureRM.NetCore | Select Name, Version

Name            Version
----            -------
AzureRM.Netcore 0.11.0
AzureRM.Netcore 0.12.0
```

### Module Version

See above Repo section
```powershell
Get-Module -Name AzureRM -ListAvailable
```

### Environment Data

```powershell
PS Azure:\> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      6.1.0-preview.3
PSEdition                      Core
GitCommitId                    v6.1.0-preview.3
OS                             Linux 4.4.0-130-generic #156-Ubuntu SMP Thu Jun 14 08:53:28 UTC 2018
Platform                       Unix
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0


Azure:/
PS Azure:\>
```

### Debug Output

```

```

**Comment by maddieclayton** [comment]
@MaximoTrinidad Have you installed the AzureRM.Netcore module on CloudShell in a previous session?  When I log onto a clean version of CloudShell I am only seeing 0.12.0.  It might be the case that you manually installed AzureRM.Netcore version 0.11.0, and so this persists in new sessions.

To remove the older version, you can run this:
```
Get-ChildItem -Path /usr/local/share/powershell/Modules/ -Recurse -Filter "*0.11.0*" | Remove-Item
```

**Comment by MaximoTrinidad** [comment]
Great!!  Your suggestion help pin-point where I got the issue.

First, I was under the understanding, when Microsoft refresh the Cloud Shell, I thought everything was rebuild (including modules).

Well, apparently I did manually installed version AzureRM.Netcore v 0.11.0 in one point in time. This was installed on my local usr module folder: "/home/maximo/.local/share/powershell/Modules".

Get-Module helped to identifying the folder, and then I modify your oneliner to remove it:
```
Get-ChildItem /home/maximo/.local/share/powershell/Modules/ -Recurse -Filter"AzureRM.*" | Remove-Item
```
It works!

![azurerm_00_2018-07-19_17-05-54](https://user-images.githubusercontent.com/19892435/42970641-d836442e-8b77-11e8-9ca5-a1b9d769a2f4.jpg)

![azurerm_01_2018-07-19_17-05-54](https://user-images.githubusercontent.com/19892435/42970552-952cce8c-8b77-11e8-83ce-b72973b7cbb5.jpg)

Thanks for wake up call!
:)
